src/main/java/mate/academy/hibernate/relations/service/impl/CountryServiceImpl.java

```diff
@@ -1,16 +1,23 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import jakarta.persistence.EntityNotFoundException;
+import mate.academy.hibernate.relations.dao.CountryDao;
+import mate.academy.hibernate.relations.dao.impl.CountryDaoImpl;
 import mate.academy.hibernate.relations.model.Country;
 import mate.academy.hibernate.relations.service.CountryService;
+import mate.academy.hibernate.relations.util.HibernateUtil;
 
 public class CountryServiceImpl implements CountryService {
+    private final CountryDao countryDao = new CountryDaoImpl(HibernateUtil.getSessionFactory());
     @Override
     public Country add(Country country) {
-        return null;
+        return countryDao.add(country);
     }
 
     @Override
     public Country get(Long id) {
-        return null;
+        return countryDao.get(id).orElseThrow(
+                () -> new EntityNotFoundException("Couldn't get a country from DB.")
```

Comment:
The exception message in the `orElseThrow` method should be more specific. Include the ID of the country that was not found to aid in debugging. For example: `() -> new EntityNotFoundException("Couldn't get country with ID " + id + " from DB.")`